packages/patternlab/styleguide/source/_patterns/01-atoms/09-media/site-logo.twig

```diff
@@ -1,6 +1,6 @@
 <div class="ma__site-logo">
   <a href="/" title="Mass.gov home page">
-    <img src="{{ sealImage }}" alt="" width="45" height="45" />
+    <img src="{{ sealImage }}" alt="Massachusetts State Seal" width="45" height="45" />

```

Comment:
This alt is intentionally left blank for the Mass.gov site title in the header.  In this context, the component has the text content of "Mass.gov" as the site title and screen reader users wouldn't get much of helpful information from "Massachusetts State Seal."  So, the image is treated as decorative.

Reply:
I see, removed